### PR TITLE
Cdar 774/880 time sync script post process

### PR DESCRIPTION
**Pull request by MishkaMN**
# PR Details
## Description

Adding post process feature to the time sync checking script for cdasim.
It also reverts a bug introduced in previous PR: https://github.com/usdot-fhwa-stol/carma-analytics-fotda/pull/63
where many of the automatic script checking was turned off

## Related GitHub Issue
https://github.com/usdot-fhwa-stol/carma-analytics-fotda/issues/61

## Related Jira Key
https://usdot-carma.atlassian.net/browse/CDAR-880

## Motivation and Context
vru master release prep

## How Has This Been Tested?
sim pc 1

## Types of changes

- [ ] Defect fix (non-breaking change that fixes an issue)
- [X] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that cause existing functionality to change)

## Checklist:

- [X] I have added any new packages to the sonar-scanner.properties file
- [X] My change requires a change to the documentation.
- [X] I have updated the documentation accordingly.
- [X] I have read the [**CONTRIBUTING**](https://github.com/usdot-fhwa-stol/carma-platform/blob/develop/Contributing.md) document.
- [X] I have added tests to cover my changes.
- [X] All new and existing tests passed.

### CODE REVIEW

**Comment by MishkaMN** [comment]
Please just approve if ready, but don't merge. Thanks! @Saeid-Solei

**Review by Saeid-Solei — APPROVED** [review]

**Pull request merged**
